index.js

```diff
@@ -127,7 +132,7 @@ Adapter.prototype.broadcast = function(packet, opts){
             if (ids[id] || ~except.indexOf(id)) continue;
             socket = self.nsp.connected[id];
             if (socket) {
-              socket.packet(encodedPackets, true, flags.volatile);
```

Comment:
`flags.volatile` was ignored. Actually we had to set flags like `socket.volatile`.